### PR TITLE
Switch main to 6.1 and add automated refresh to v1.6 branch

### PR DESCRIPTION
**Pull request by davidcassany**
This is an alternate approach for #23

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-    fileName: "sl-micro-60-baremetal"
-    displayName: "SL Micro 6.0"
-    osRepo: registry.suse.com/suse/sl-micro/6.0/baremetal-os-container
-    isoRepo: registry.suse.com/suse/sl-micro/6.0/baremetal-iso-image
-    limit: 3
+    fileName: "sl-micro-61-baremetal"
+    displayName: "SL Micro 6.1"
+    osRepo: registry.suse.com/suse/sl-micro/6.1/baremetal-os-container
+    isoRepo: registry.suse.com/suse/sl-micro/6.1/baremetal-iso-image
+    limit: 1
```
Comment (davidcassany) [inline config.yaml:7]: Only one because we want to publish GM and once this is out keep adding on top, but not pre-GM

**Review by davidcassany — COMMENTED** [review]
Note that if we merge this we must create a new `v1.6` branch from main.

**Review by anmazzotti — CHANGES_REQUESTED** [review]
All looks good, but may you run the script locally to populate the json files in this PR already? I see the file names changed, but the content is still the same and includes the 6.0 images.

**Comment by davidcassany** [comment]
Rebased and updated with a refresh execution mually

**Comment by anmazzotti** [comment]
Nice! Thank you. 
I see that 2 ISO images ended up included, for v2.1.2 and v2.2.2, the limit is 1 per minor version. 
Not sure this is desired. If we don't want to publish the v2.1.2 images then I would remove them from the registry. Alternatively we can implement some sort of filter in the script.
Anyway the PR is good to me now.

**Review by anmazzotti — APPROVED** [review]

**Comment by davidcassany** [comment]
Agree, there will be a follow up PR to arrange the glitch with the ISO image es.

**Pull request merged**